models/dave_worlds/worlds/dave_graded_seabed.world

```diff
@@ -80,5 +80,12 @@
 
     <plugin name="sc_interface" filename="libuuv_sc_ros_interface_plugin.so"/>
 
+    <gui fullscreen='0'>
+      <camera name='user_camera'>
+        <pose frame=''>14.26 -12.53 0.054 -0.0 0.28 2.36</pose> 
```

Comment:
So, I'm having second thoughts (of course I didn't try this task myself before asking you to do it as a first practice PR). I'd like a second opinion from @dtdavi1.

@dtdavi1 For context, this PR (that Dharini is practicing with as her first PR) is adding a default camera angle as a small user experience improvement, because we realized that when a new user launches the world, they see the seabed, but they don't see the sensor as described in the tutorial. So they might (and did) go through troubleshooting to "fix" why their sensor isn't showing up. We thought adding a default camera angle closer to the camera would help.

Now taking a look at how things are loaded, the camera pose numbers here are inevitably very specific, while the sensor is loaded in a separate launch file (`upload_teledyne_whn.launch`), not in this world file. So hardcoding the numbers in this file is not the best style. While this world file is currently only used by the two DVL launch files, there's nothing preventing one from writing a launch file in the future and using this world file for something completely different from the DVL, which would make these camera angles obsolete for that use case. 

I might even say pass it in from the top-level launch file, but that would unnecessarily complicate things.

Maybe a simpler fix is a "tutorial improvement" as opposed to "code improvement"?

Reply:
I don't have strong feelings about it.  The camera isn't positioned well for some of the demos (it's not even close in the DVL demos) that reuse world files from other demos.  If the world in question isn't being used by any other demos anymore, it might be easiest just to move the camera in the world file.

Reply:
Thanks for the input! Okay, guess we'll keep this here.

I would suggest though @quarkytale that a comment be added right above the `<pose>` line, that this pose is meant to be for the DVL demos - and name the two launch files. That way, if in the future someone does decide to reuse this world for another launch file, they'll see this and refactor the camera instead of changing it and breaking the DVL demos.

Reply:
Updated as per your suggestion @mabelzhang!